### PR TITLE
Create Activision.xml

### PR DESCRIPTION
**Pull request by davidabian**

### CODE REVIEW

**Comment by J0WI** [comment]
I got some more:
press.activision.com
jobs.activision.com
investor.activision.com
community.activision.com
eupcareers.activision.com
ambassador.activision.com

**Comment by davidabian** [comment]
Thanks, @J0WI! But no checks are running. :cry:

**Comment by J0WI** [comment]
trigger tests

**Comment by J0WI** [comment]
I fear some of them won't work, I didn't test all of them myself.

File: src/chrome/content/rules/Activision.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!--
+	Not supporting HTTPS or using an invalid certificate:
```
Comment (J0WI) [inline src/chrome/content/rules/Activision.xml]: Could you also describe what kind of error (refused, mismatch, timeout ...)

**Comment by jeremyn** [comment]
@davidabian Are you still interested in working on this one?

**Comment by J0WI** [comment]
It seems like `sa.activision.com` is no longer available.

**Comment by davidabian** [comment]
Thanks. I hope this ruleset is now correct.

**Comment by J0WI** [comment]
Yes, thank you!

**Pull request merged**